lib/update_school_data.rb

```diff
@@ -28,6 +28,17 @@ class UpdateSchoolData
     town: 'Town',
   }
 
+  READABLE_PHASE_MAPPINGS = {
```

Comment:
Is this the same that has been defined in the school model?

Reply:
Yes, I guess I was being lazy 😂 . How do I access that in /lib?